frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/block-pool.ts

```diff
@@ -38,24 +40,31 @@ export enum Actions {
   created = 'created',
   failed = 'failed',
   notAllowed = 'notAllowed',
+  bound = 'bounded',
 }
 
-export const blockPoolFooter = (action: string) => {
+export const verifyFooterActions = (action: string) => {
   switch (action) {
     case Actions.failed:
-      cy.byTestID('empty-state-body').contains(poolMessage[POOL_PROGRESS.FAILED]);
+      cy.log('Check try-again-action and finish-action are enabled');
       cy.byLegacyTestID('modal-try-again-action').should('be.visible');
       cy.byLegacyTestID('modal-finish-action').click();
       break;
     case Actions.created:
-      cy.byTestID('empty-state-body').contains(poolMessage[POOL_PROGRESS.CREATED]);
```

Comment:
I think you also want to update this in create block pool tests.

Reply:
i already added https://github.com/openshift/console/pull/8661/files#diff-bd3770a2c52709f48a1c79ce1e39d1014a1f5591e40b95521bb1faafcd91dbc0R33

And this check only for pool creation modal, not for pool creation page

Reply:
okay